src/classes/error-message.ts

```diff
@@ -123,7 +143,7 @@ export class ErrorMessage {
    * @param {String} message - the error message
```

Comment:
Can we use this opportunity to also remove jsdoc type annotations?

Reply:
There are a lot of jsdoc annotations.  I think it would be best to have a specific PR for removing the annotations.

Reply:
👍